NewForce-Cohort-9/tabloid-full-stack-code-crusaders
# finised category list

# Description
Added feature to view a list of Categories to project. [Trello Ticket #9](https://trello.com/c/wy81zAjS)

# Testing Instructions

1. git add and git commit whatever you're working on on your branch
2. git checkout main
3. git fetch --all
4. git checkout
5. If you don't see the most recent version of my files, git pull origin
6. If you haven't created the Tabloid database, please do so.
7. Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-code-crusaders/blob/main/SQL/01_Tabloid_Create_DB.sql)
8. Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-code-crusaders/blob/main/SQL/02_Tabloid_Seed_Data.sql)
9. Click the execute with debugger solid green button to launch the app.
10. In browser, please go to Login on the nav bar and log in with this email: foo@bar.com with any password of your choosing.
11. This should redirect you back to the home page.
12. Click the Category Management link on nav bar
13. You should see a list of Categories in alphabetical order. The edit and delete buttons do not have functionally right now.

Cooking
Cthulhu Sightings
Entertainment
History
Home and Garden
Music
Politics
Science
Technology

# Checklist
